src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/AutowireTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator\Traits;
+
+trait AutowireTrait
```

Comment:
Shouldn't all those traits be marked as `@internal`. They don't seem to be useful for others.

Reply:
I wondered the same, but decided not because there is no benefit doing so imho: the implementation in the traits are simple, their use is simple also and the methods they expose will be not "internal", so the annotation could be confusing in fact.